flexcraft/pipelines/binder.py

```diff
@@ -150,14 +151,21 @@ def parse_target(c, path, num_aa):
     prev_threshold=0.9,
     use_cycler="False",
     fix_template="False",
+    af2_cycler_repeats=10, # how many iterations the af2_cycler should do
     visualize_centers="False",
     ipae_shortcut_threshold=0.35, # which ipae threshold to use for Rosetta relaxation
-    save_fail_pdbs="True", #whether to save pdb files of failed designs
-    allow_target_mutations=0,
+    allow_target_mutations="0", # Number of target interface residues allowed to be redesigned by ProteinMPNN, pass as STR. # int (e.g. "0") or list giving range, (e.g. "[0,5]"" for up to 5 mutations)
     redesign_radius=10, # Target residue distance for redesign in Angstrom (if allow_target_mutations > 0). 
+    save_af2cycler_PDB="True", # whether to save all af2cycler PDB files
```

Comment:
Please use lower-case "pdb" for all names that are currently named "something_PDB"